<launch>
  <arg name="veh"/>
  <arg name="map_file" default="hub"/>
  <group ns="$(arg veh)">
    <remap from="/robot_name/camera_node/image/compressed" to="/$(arg veh)/camera_node/image/compressed"/>
        <remap from="/robot_name/camera_node/camera_info" to="/$(arg veh)/camera_node/camera_info"/>
    <node pkg="augmented_reality_apriltag" type="augmented_reality_apriltag.py" name="augmented_reality_apriltag" output="screen">
    </node>
  </group>

</launch>